tests/bears/BearTest.py

```diff
@@ -1,26 +1,25 @@
-from collections import defaultdict
 import datetime
-from io import BytesIO
 import multiprocessing
+import shutil
 import unittest
+from collections import defaultdict
+from io import BytesIO
 from os.path import abspath, exists, isfile, join, getmtime
-import shutil
-
-from freezegun import freeze_time
 
 import requests
 import requests_mock
 import urllib3
+from freezegun import freeze_time
 
 from coalib.bearlib.aspects.collections import AspectList
 from coalib.bearlib.aspects.Metadata import CommitMessage
-from coalib.bears.Bear import Bear
 from coalib.bears.BEAR_KIND import BEAR_KIND
+from coalib.bears.Bear import Bear
 from coalib.bears.GlobalBear import GlobalBear
 from coalib.bears.LocalBear import LocalBear
-from coalib.results.Result import Result
 from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
 from coalib.processes.communication.LogMessage import LogMessage
+from coalib.results.Result import Result
```

Comment:
are you just sorting imports here? That's unrelated, at least those could go in an extra commit easily; this one is so big that I'd appreciate it if at least unrelated stuff would be in a new commit